### DISCUSSION TITLE
프로젝트 폴더 구조

### DISCUSSION THREAD

**Discussion by jayjeong8** [EasyAndBeauty/DailyReceipt-Flutter#15] [Ideas]
프로젝트에 사용할 폴더 구조를 논의합니다.

@gygy7151 폴더 구조 관련해서 여기서 논의해요~!

**Comment by jayjeong8** [comment]
#8 작업하면서 `libs/` 하위에 `models/`, `screens/` 만 추가해놓을게요! 
model 파일에서 [예시](https://codelabs.developers.google.com/codelabs/flutter-app-testing#3)처럼 model이랑 provider를 같이 관리하고, 규모가 커지면 model이랑 provider를 각각의 경로로 분리하면 좋을 것 같아요!!
<img width="99" alt="image" src="https://github.com/EasyAndBeauty/DailyReceipt-Flutter/assets/82160479/bd01eaab-e1ae-4597-8828-cc0288d2d818">

@gygy7151 @IGhost-P

**Reply by IGhost-P** [reply]
좋은 방법입니다..! 제가 아직 플러터 폴더구조가 익숙하지 않앗는데 ㅠㅠ 감사해용
